vllm/model_executor/models/gpt_bigcode.py

```diff
@@ -49,10 +49,11 @@ def __init__(self, config: GPTBigCodeConfig):
         super().__init__()
         self.hidden_size = config.hidden_size
         total_num_heads = config.num_attention_heads
-        tensor_model_parallel_world_size = (
+        self.tensor_model_parallel_world_size = (
             get_tensor_model_parallel_world_size())
-        assert total_num_heads % tensor_model_parallel_world_size == 0
-        self.num_heads = total_num_heads // tensor_model_parallel_world_size
+        assert total_num_heads % self.tensor_model_parallel_world_size == 0
+        self.num_heads = \
+            total_num_heads // self.tensor_model_parallel_world_size
```

Comment:
Nit: Please use parenthesis for line continuation according to google python style guide.
```suggestion
        self.num_heads = (
            total_num_heads // self.tensor_model_parallel_world_size)
```